modules/calibration/data/dev_kit_advanced_ne-s/lidar_dag/lidar.dag

```diff
@@ -0,0 +1,27 @@
+# Define all coms in DAG streaming.
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/robosense/librobosense_driver_component.so"
+
+    components {
+      class_name : "RobosenseComponent"
+      config {
+        name : "RS16_driver"
+        config_file_path : "/apollo/modules/drivers/lidar/conf/rs16.pb.txt"
+      }
+    }
+}
+
+module_config {
+    module_library : "/apollo/bazel-bin/modules/drivers/velodyne/compensator/libvelodyne_compensator_component.so"
+
+    components {
+      class_name : "CompensatorComponent"
+      config {
+        name : "velodyne_compensator"
+        config_file_path : "/apollo/modules/drivers/lidar/conf/velodyne16_fusion_compensator.pb.txt"
+        readers {channel: "/apollo/sensor/lidar16/PointCloud2"}
```

Comment:
rename velodyne16_fusion_compensator.pb.txt to lidar16_fusion_compensator.pb.txt , be sure to edit this in lidar.dag too